src/Maths/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+    <PropertyGroup>
+        <SilkDescription>A small and portable mathematics and linear algebra library, useful in graphics and compute workloads.</SilkDescription>
+        <SilkExtendedDescription>This package has no other Silk.NET dependencies, so feel free to use it any project in any workload regardless of whether it uses other Silk.NET libraries or not. This is akin to System.Numerics.Vectors, only with extra types and generics (so any numerical type can be used rather than just float).</SilkExtendedDescription>
```

Comment:
Please add something like "Supports Half, float, double, decimal, byte, sbyte, ushort, short, uint, int, ulong, long, Complex and BitInteger" just for posterity

Reply:
Done.